### PR TITLE
Post refresh: Remove usage of `theme-classic`

### PR DESCRIPTION
**Pull request by umpox**

### CODE REVIEW

File: client/web/src/user/settings/profile/UserSettingsProfilePage.tsx [hunk 1]
```diff
@@ -81,8 +81,8 @@ export const UserSettingsProfilePage: React.FunctionComponent<Props> = ({
                 className="user-settings-profile-page__heading"
             />
             {props.activation?.completed && percentageDone(props.activation.completed) < 100 && (
-                <Container className="user-settings-profile-page__activation-container mb-3">
-                    <div className="user-settings-profile-page__activation-header">
+                <Container className="mb-3">
+                    <div>
```
Comment (eseliger) [inline client/web/src/user/settings/profile/UserSettingsProfilePage.tsx:85]: This `div` is not required anymore, is it?

**Review by eseliger — APPROVED** [review]

File: client/web/src/user/settings/profile/UserSettingsProfilePage.tsx (hunk 1 above)
Comment (umpox) [reply client/web/src/user/settings/profile/UserSettingsProfilePage.tsx:85]: Nope (I hope not 😁 )

File: client/web/src/user/settings/profile/UserSettingsProfilePage.tsx [hunk 2]
```diff
@@ -80,12 +80,10 @@ export const UserSettingsProfilePage: React.FunctionComponent<Props> = ({
                 }
                 className="user-settings-profile-page__heading"
             />
-            {props.activation?.completed && percentageDone(props.activation.completed) < 100 && (
```
Comment (valerybugakov) [inline client/web/src/user/settings/profile/UserSettingsProfilePage.tsx:83]: Curious why `percentageDone(props.activation.completed) < 100` is no longer needed here?

**Review by valerybugakov — APPROVED** [review]

File: client/web/src/user/settings/profile/UserSettingsProfilePage.tsx (hunk 2 above)
Comment (umpox) [reply client/web/src/user/settings/profile/UserSettingsProfilePage.tsx:83]: Oops, you didn't see this 😂

**Pull request merged**